docs/platform/release-notes/tizen-5-5-m2.md

```diff
@@ -0,0 +1,641 @@
+  - New AudioChannel and SampleType of AudioIO have been added.
+  - New ImageUtil API set to extract thumbnail has been added.
+  - New MediaPlayer API set to export video frames and audio PCM data has been added.
+  - New MediaPlayer API set to cancel preparing has been added.
+  - New MediaPlayer API set for audio codec and offload has been added.
+  - New Multimedia Remote API set for subtitle, 360 mode has been added.
+  - New Vision API set for detection and classification using inference engine has been added.
+  - New UI component set of NUI has been added.
+  - Tizen.NUI.UIComponents module has been deprecated.
+  - New NUI API set for window creation and rotation has been added.
+  - New SystemSettings API set to get rotary event is enabled has been added.
+  - New VoiceControlManager API set has been added.
+
+### Known Issues
+
+- Xamarin.Forms
+  - For more information on the list of limitations, see [here](https://developer.tizen.org/development/api-reference/.net-application/current-xamarin.forms-limitations).
+
+
+## Toolchain
+
+### New and changed features
+
+- Toolchain watermarking (annobin) function has been added.
+- AddressSanitizer shadow scaling option has been applied to reduce runtime overhead on RAM.
+- Libasansi library has been introduced to support dotnet ASan runtime.
+- “gcov-force-options” package (scripts to enable automatic package coverage instrumentation) has been added.
+
+### Fixes
+
+- GCC
+  - Bugfix on ICE (Internal Compiler Error) has been back-ported from upstream - bug #83623 #85496.
+  - LeakSanitizer allocator bug on 64-bit target (aarch64, x86_64) has been fixed.
+  - IntegerSanitizer option's bug with "-fsanitize=shift,unsigned-integer-overflow" has been fixed.
+  - AddressSanitizer runtime option has been modified (allocator_may_return_null=1) to provide better report.
+- GLIBC
+  - fclose SIGABORT issue on multi-thread environment with race condition has been fixed by applying bug #15142.
+  - New localedata(tl_PH) has been added (bug #15260).
+
+
+## Machine Learning
+
+### New and changed features
+
+- Inference API set
+  - Neural network pipeline API set, which allow to construct and execute data stream pipelines with neural networks, tensor operators, input nodes, and multimedia filters, have been added.
+  - Neural network single API set, which allow to invoke neural network models with simple API calls, have been added.
+- NNStreamer
+  - NNStreamer (https://github.com/nnsuite/nnstreamer) 1.0 has been added as a set of gstreamer plugins to enable Inference API set.
+  - NNStreamer in Tizen 5.5 M2 supports tensorflow-lite (1.13) models, custom C/C++ models (.so files), and python codes. Such models may be supplied by applications as resource files.
+- nnfw: Neural Network Runtime
+  - Introducing the ‘nn package’ to support the direct loading of TensorFlow lite models.
+  - In order to use the ‘nn package’ independently, a new Runtime API set have been introduced. (Experimental)
```

Comment:
In order to use `nn package` independently, a new Runtime API set has been introduced. (Experimental)

Reply:
Will accept. Thank you!